.github/CODEOWNERS

```diff
@@ -16,13 +16,19 @@
 
 * @department-of-veterans-affairs/va-platform-cop-frontend
 
+
+# Platform Design System
+
 src/applications/_mock-form @department-of-veterans-affairs/platform-design-system-fe
 src/applications/burial-poc-v6 @department-of-veterans-affairs/platform-design-system-fe
 src/platform/forms        @department-of-veterans-affairs/platform-design-system-fe @department-of-veterans-affairs/platform-va-product-forms
 src/platform/forms-system @department-of-veterans-affairs/platform-design-system-fe @department-of-veterans-affairs/platform-va-product-forms
+src/applications/simple-forms @department-of-veterans-affairs/platform-va-product-forms
+src/platform/static-pages/simple-forms @department-of-veterans-affairs/platform-design-system-fe @department-of-veterans-affairs/platform-va-product-forms
 script/component-migration @department-of-veterans-affairs/platform-design-system-fe
 src/applications/ds-playground @department-of-veterans-affairs/platform-design-system-fe
 src/applications/ds-v3-playground @department-of-veterans-affairs/platform-design-system-fe
+src/applications/static-pages/wizard @department-of-veterans-affairs/platform-design-system-fe
```

Comment:
If this is forms related, should it be @department-of-veterans-affairs/platform-va-product-forms?

Reply:
Good question - does the DST also need to review? Or `platform-va-product-forms` only?

Reply:
At the moment I think DST is just tagged for visibility for platform form directories but the forms team is generally handling PR reviews related to them. I think it's ok to have both if `/static-pages/wizard` is a platform related form directly but I don't actually know what that is.

Reply:
👍  I'll change it to forms team only; that seems to make more sense.